courses/beginner/10.md

```diff
@@ -48,207 +41,183 @@ control 'nginx-modules' do
-control 'second-overlay-control'
-control 'third-overlay-control'
-control 'fourth-overlay-control'
-control 'fifth-overlay-control'
+require_controls 'my_nginx' do
+  control 'nginx-modules' do
+    impact 0.5
+  end
+  control 'nginx-conf-file'
+end
 ```
 
-Whenever `my_nginx` is executed, in addition to its own controls, it will run only the controls specified in the `require_controls` block. In the case, the following controls would be executed:
-
- - nginx-version
- - nginx-modules
- - nginx-conf
- - first-overlay-control
- - third-overlay-control
-
-Controls `second-overlay-control`, `fourth-overlay-control`, and `fifth-overlay-control` would not be run, just as if they were manually skipped. This method of including specific controls ensures only the controls specified are executed; if new controls are added to a later version of `my_nginx_overlay`, they would not be run.
+:::
 
-And, just the way its possible to modify controls when using `include_controls`, controls can be modified as well.
+In the above example, all included or required controls from `my_nginx` profile are executed. However, should control `nginx-modules` fail, it will be raised with an impact of `0.5` instead of the originally-intended impact of `1.0`.
 
+#### Inputs with profile inheritance
 
-```ruby
-# my_nginx profile  
-control 'nginx-version' do
-  impact 1.0
-  title 'NGINX version'
-  desc 'The required version of NGINX should be installed.'
-end
+In order for a control to run properly, the values for inputs can be overriden to align with your environment. Let's say your environment uses `root` not `admin`, so the input for `nginx-shell-access` control can be updated to measure accordingly.
 
-control 'nginx-modules' do
-  impact 1.0
-  title 'NGINX modules'
-  desc 'The required NGINX modules should be installed.'
-end
+One way of updating the input is by using the `inspec.yml` file:
 
-control 'nginx-conf' do
-  impact 1.0
-  title 'NGINX configuration'
-  desc 'The NGINX config file should owned by root, be writable only by owner, and not writeable or and readable by others.'
-end
-
-require_controls 'my_nginx_overlay' do
-  control 'first-overlay-control' do
-    impact 0.5
-  end
-  control 'third-overlay-control'
-end
+::: code-tabs
+@tab my_nginx
+```yaml
+inputs:
+  - name: admin_users
+    type: array
+    value:
+      - 'admin'
 ```
 
-```ruby
-# my_nginx_overlay
-control 'first-overlay-control' do
-  impact 1.0
-end
-
-control 'second-overlay-control'
-control 'third-overlay-control'
-control 'fourth-overlay-control'
-control 'fifth-overlay-control'
+@tab my_nginx_overlay
+```yaml
+inputs:
+  - name: admin_users
+    type: array
+    value:
+      - 'root'
+    profile: my_nginx
 ```
+:::
+
 
-As with the prior example, only `first-overlay-control` and `third-overlay-control` are executed, but if `first-overlay-control` fails, it will report with an impact of `0.5` instead of the originally-intended `1.0` impact.
+::: danger Overriding Inputs
```

Comment:
Excited for the first use of the danger tab

Reply:
We decided to remove it :/